jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java

```diff
@@ -1,34 +1,125 @@
 package io.github.epam.angular.tests.elements.complex;
 
+import io.github.com.pages.sections.RadioButtonSection;
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import static io.github.com.StaticSite.angularPage;
-import static io.github.com.pages.AngularPage.basicRadioButtons;
-import static io.github.com.pages.AngularPage.firstBasicRadioButton;
-import static io.github.com.pages.AngularPage.secondBasicRadioButton;
+import static io.github.com.pages.AngularPage.*;
 import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.apache.tika.mime.MediaType.text;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.matchesRegex;
 import static org.testng.Assert.assertEquals;
 
 public class RadioButtonTests extends TestsInit {
 
+    private static String enableButton = ".* cdk-focused cdk-mouse-focused mat-radio-checked";
+    private static String disableSeasonButton = "mat-radio-button example-radio-button mat-accent";
+    private static String disableBasicButton = "mat-radio-button mat-accent";
+
```

Comment:
Please remove all double blank lines in classes. After formatting, there should be only one blank line between code blocks.